src/v2/providers/eventarc.ts

```diff
@@ -54,14 +54,26 @@ export interface EventarcTriggerOptions extends options.EventHandlerOptions {
   filters?: Record<string, string>;
 }
 
+/**
+ * A function that takes a CloudEvent.
+ */
 export type CloudEventHandler = (event: CloudEvent<any>) => any | Promise<any>;
 
-/** Handle an Eventarc event published on the default channel. */
+/** Handles an Eventarc event published on the default channel.
+ * @param eventType - Type of the event to trigger on.
+ * @param handler - A function to execute when triggered.
+ * @returns A Cloud Function that you can export and deploy.
+ */
 export function onCustomEventPublished<T = any>(
   eventType: string,
   handler: CloudEventHandler
 ): CloudFunction<CloudEvent<T>>;
 
+/** Handles an Eventarc event.
+ * @param opts - Options to set on this Cloud Function
```

Comment:
I strongly prefer just "function," and think "Cloud Functions" -- the capped product -- should always be plural.